claat/render/md.go

```diff
@@ -174,7 +174,7 @@ func (mw *mdWriter) image(n *types.ImageNode) {
 	}
 
 	if n.Title != "" {
-		mw.writeString(fmt.Sprintf("title=\"%q\" ", n.Title))
+		mw.writeString(fmt.Sprintf("title=\"%s\" ", n.Title))
```

Comment:
Does the fix still work if it's `title=%q` instead?

Reply:
no, stripping quotes in the parser only corrects for author-added quotes, but this line in the renderer is incorrect by itself because the `%q` symbol in Go adds quotes, so `title=\"%q\" ` will always render `title=""some title""`

Reply:
yes and that's cleaner, I'll switch over to that